src/main/java/hu/eszter/bokkon/model/participants/Farmer.java

```diff
@@ -0,0 +1,67 @@
+package hu.eszter.bokkon.model.participants;
+
+import hu.eszter.bokkon.model.animal.Animal;
+
+import java.util.*;
+
+public class Farmer implements MoveAnimal {
+
+    private final String name;
+    private final Random random = new Random();
+    private Map<Animal, Integer> farmerLiveStock = new HashMap<>();
+
+    public Farmer(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Map<Animal, Integer> getFarmerLiveStock() {
+        return farmerLiveStock;
+    }
+
+    @Override
+    public void addAnimal(Animal animal) {
+        farmerLiveStock.putIfAbsent(animal, 0);
+        farmerLiveStock.put(animal, farmerLiveStock.get(animal) + 1);
```

Comment:
In general, computeIfPresent is shorter than `put(key, get(key) + x)`
`map.computeIfPresent("key",(k,v)->v + x);`